Chore/semaphore v3 (#18)
semaphore-protocol/boilerplate: README.md

> **Warning**  
> The group id is a number!

> #### Steps to follow every time the smart contract is deployed:

Reviewer: ```suggestion

```
The previous list could continue.

Author: True

> 1. Change the `CONTRACT_ADDRESS` (with the new contract address) and `GROUP_ID` (with the group id) in the `.env` file.
> 

Reviewer: ```suggestion
```

This should already be explained in the previous point (?).

> 2. Change the `address` (with the new contract address) and `startBlock` (with the block number of the transaction where the contract was created) in the `subgraph.yaml` file.
> 
> 3. Deploy the subgraph again.
> 
> 4. Copy the `Feedback.sol` file (contract artifacts) from `apps/contracts/build/contracts/contracts/Feedback.sol` folder to `apps/subgraph/contract-artifacts` and `apps/web-app/contract-artifacts `folders manually or in the root folder, run: `yarn copy:contract-artifacts` to do it automatically.

Reviewer: ```suggestion
3. Change the `address` (with the new contract address) and `startBlock` (with the block number of the transaction where the contract was created) in the `apps/subgraph/subgraph.yaml` file.

4. Copy your contract artifacts from `apps/contracts/build/contracts/contracts` folder to `apps/subgraph/contract-artifacts` and `apps/web-app/contract-artifacts `folders manually. Or run `yarn copy:contract-artifacts` in the project root to do it automatically.

5. Deploy the subgraph again.
```

The subgraph should be deployed only after updating the contract artifacts.
